Islands.java

```diff
@@ -0,0 +1,70 @@
+/**
+* You are given a 2-dimensional map of tiles. 
+* Each tile is either land or water. 
+* You have to write a function that counts the number of islands.
+*
+* @author  Argyro Gounari
+* @version 1.0
+* @since   01-07-2018 
+*/
+public class Islands {
+
+	/**
+	 * Function that counts the number of islands.
+	 * False -> Water
```

Comment:
Have constants for water and land, then you don't even need to read the comments to understand the code.